app/services/queries/cclow/legislation_query.rb

```diff
@@ -48,7 +49,13 @@ def filter_by_tags
       def filter_by_from_date
         return scope unless params[:from_date].present?
 
-        scope.where('updated_at >= ?', params[:from_date])
+        scope.where('updated_at >= ?', Date.new(params[:from_date].to_i, 1, 1))
```

Comment:
I think the logic here will be more complicated unfortunately, not updated_at date

Reply:
I'm not sure what's the logic behind time filter, will try to find out.